sys-libs/ncurses/ncurses-6.2-r1.ebuild

```diff
@@ -270,24 +278,11 @@ multilib_src_install() {
 }
 
 multilib_src_install_all() {
-#	if ! use berkdb ; then
+	if ! use symlink-usr ; then
 		# We need the basic terminfo files in /etc for embedded/recovery. #37026
 		einfo "Installing basic terminfo files in /etc..."
-		local terms=(
-			# Dumb/simple values that show up when using the in-kernel VT.
-			ansi console dumb linux
-			vt{52,100,102,200,220}
-			# [u]rxvt users used to be pretty common.  Probably should drop this
-			# since upstream is dead and people are moving away from it.
-			rxvt{,-unicode}{,-256color}
-			# xterm users are common, as is terminals re-using/spoofing it.
-			xterm xterm-{,256}color
-			# screen is common (and reused by tmux).
-			screen{,-256color}
-			screen.xterm-256color
-		)
 		local x
```

Comment:
How about adding a short comment, like `Flatcar: Add a symlink-usr USE flag for keeping a minimal set of terminfo files in /usr/share/terminfo.`?
I know, the original coreos commit did not have such comments, but recently Flatcar started writing comments like that, so other folks could easily recognize it is Flatcar-specific code.

Reply:
I agree. I will add it.